### ISSUE TITLE
Documentation for the transaction feature

### ISSUE THREAD

**Issue by davidballester** [Vermonster/fhir-kit-client#62]
## Expected Behavior
The [documentation examples for the transaction function](https://vermonster.github.io/fhir-kit-client/fhir-kit-client/0.3.1/module-fhir-kit-client-Client.html#transaction) should wrap the `requestBundle` into an object with a `body` property. Or even better, remove the need for such wrapping.

```javascript
const requestBundle = {
    ...
};

fhirClient.transaction({
    body: requestBundle,
});
```

Also, the `params` argument states the `body` is of type `String`, but an object is accepted.

## Actual Behavior
The transaction function requires an argument with a `body` property containing the transaction resource. However, the [documentation examples](https://vermonster.github.io/fhir-kit-client/fhir-kit-client/0.3.1/module-fhir-kit-client-Client.html#transaction) shows a `requestBundle` used directly as an argument.

## Steps to Reproduce the Problem

  1. [Documentation issue](https://vermonster.github.io/fhir-kit-client/fhir-kit-client/0.3.1/module-fhir-kit-client-Client.html#transaction)

## Specifications

  - Version: `0.3.1`
  - Platform:
  - Subsystem:

**Comment by davidballester** [comment]
I've created a [PR#63](https://github.com/Vermonster/fhir-kit-client/pull/63) to unwrap the parameter from the body property.

**Comment by Jammjammjamm** [comment]
The current implementation is what is intended, so we'd prefer to update the documentation to reflect that. This avoids making a breaking change and keeps `transaction` consistent with `batch`, `create`, and `update`. If you don't want to update your PR, we'd be happy to update to documentation ourselves.

**Comment by davidballester** [comment]
OK, I hadn't seen the other features! Now, I've created [PR#64](https://github.com/Vermonster/fhir-kit-client/pull/64) to update `transaction` and `batch` docs examples. Let me know if that's OK!

**Comment by Jammjammjamm** [comment]
Resolved by #64
